pyproject.toml

```diff
@@ -1,6 +1,219 @@
-# build-system taken from https://github.com/pypa/setuptools_scm/#pyprojecttoml-usage
+[project]
+requires-python = ">=3.7"
+
+name        = "armory-testbed"
+description = "Adversarial Robustness Test Bed"
+dynamic     = [ "version" ]
+license     = "MIT"
+authors = [
+  { name = "Two Six Technologies", email = "armory@twosixtech.com" },
+]
+
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "License :: OSI Approved :: MIT License",
+  "Environment :: Console",
+  "Natural Language :: English",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Education",
+  "Intended Audience :: Science/Research",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Topic :: Scientific/Engineering :: Artificial Intelligence",
+]
+
+# the minimal set needed to run armory.main (launcher)
+dependencies = [
+    "wheel",
+    "setuptools_scm",
+    "docker",
+    "jsonschema",
+    "loguru",
+    "pytest",
+    "PyYAML",
+    "requests",
+    "toml",
+]
+
+[project.urls]
+Source        = "https://github.com/twosixlabs/armory"
+Documentation = "https://armory.readthedocs.io/en/latest/"
+
+[project.scripts]
+armory = "armory.__main__:main"
+
+
+# please see docs/getting_started.md#the-armory-flavors for how these
+# optional dependencies work
+[project.optional-dependencies]
+developer =[
+    "hatch", # build tools
+    "bandit[toml]", # code scanning
+    "black", # formatting
+    "black[jupyter]",
```

Comment:
`black` and `black[jupyter]` should be bound to version `22.*`. Major version changes will change how it formats.